smartsim/experiment.py

```diff
@@ -179,13 +181,23 @@ def start(self, *jobs: Job) -> tuple[LaunchedJobID, ...]:
             jobs that can be used to query or alter the status of that
             particular execution of the job.
         """
-        return self._dispatch(dispatch.DEFAULT_DISPATCHER, *jobs)
+        run_id = datetime.datetime.now().replace(microsecond=0).isoformat()
+        """Create the run id"""
```

Comment:
I think for function variables it may be more natural for the comments to come before the variable if they are going to be added.  I'm not seeing in vscode that they render as a comment

Reply:
If we are going to put these comments ahead of the variable as non-docstrings, can we make these look like traditional comments?
```suggestion
        # Create the run id
```
In theory it makes saves us a no-op instruction at runtime, but in practice I just think it looks nicer, hahaha